packages/infra/lib/api-stack.ts

```diff
@@ -1079,6 +1088,35 @@ export class APIStack extends Stack {
     return documentDownloaderLambda;
   }
 
+  private setupDocumentQueryResults(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, sentryDsn, envType, alarmAction } = ownProps;
+
+    const documentQueryResultsLambda = createLambda({
+      stack: this,
+      name: "DocumentQueryResults",
+      runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
Gotcha even though every other Lambda doesnt have the runtime?

Reply:
Not every other lambda, if you search by `createLambda(` on OSS you'll see that half don't set it. And some that do are setting for v16 - which is needed since the default one is v18.